Dockerfile

```diff
@@ -26,10 +28,11 @@ RUN cd / && CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o testground
 #::: RUNTIME CONTAINER
 #:::
 
-FROM busybox:1.31.0-glibc
+#FROM busybox:1.31.0-glibc
+
 
 RUN mkdir -p /usr/local/bin
-COPY --from=0 /testground /usr/local/bin/testground
+RUN cp /testground /usr/local/bin/testground
```

Comment:
?

Reply:
Due to `iptables`. Will bring back the two-stage build Dockerfile, before merging, this was a shortcut.